src/ajax.js

```diff
@@ -13,6 +13,7 @@ define([
 
 var
 	rhash = /#.*$/,
+	rextraQuestions = /\?+/,
```

Comment:
Not the first time we forgot to document something ;)

Not sure I understand your question but the fix (ie introduction of `??`) was for APIs that would have the callback as part of the URL, not as a parameter (see [the original ticket](http://bugs.jquery.com/ticket/4897)).